<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandleMessage</class>
 <widget class="QMainWindow" name="HandleMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>535</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>605</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>605</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="SystemMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>571</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
font-size: 12pt;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>系统消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HandleButton1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 10px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FinishedButton1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 10px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>已完成</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>265</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="FinishedButton2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 10px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>已完成</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>295</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HandleButton2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 10px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>处理</string>
    </property>
   </widget>
   <widget class="QTableView" name="UserMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
font-size: 12pt;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="BackGround">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>596</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 10px;
font-size: 20pt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EmailButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 15px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>发送邮件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CheckButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 10px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>导出</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <zorder>BackGround</zorder>
   <zorder>SystemMessage</zorder>
   <zorder>Label1</zorder>
   <zorder>line</zorder>
   <zorder>Label2</zorder>
   <zorder>UserMessage</zorder>
   <zorder>EmailButton</zorder>
   <zorder>FinishedButton2</zorder>
   <zorder>HandleButton2</zorder>
   <zorder>FinishedButton1</zorder>
   <zorder>HandleButton1</zorder>
   <zorder>CheckButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>SystemMessage</tabstop>
  <tabstop>HandleButton1</tabstop>
  <tabstop>CheckButton</tabstop>
  <tabstop>FinishedButton1</tabstop>
  <tabstop>UserMessage</tabstop>
  <tabstop>HandleButton2</tabstop>
  <tabstop>EmailButton</tabstop>
  <tabstop>FinishedButton2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
